addons/skin.estuary/xml/VideoFullScreen.xml

```diff
@@ -33,7 +33,9 @@
 		</control>
 		<control type="group" id="1">
 			<depth>DepthOSD+</depth>
-			<visible>Player.Caching</visible>
+			<visible>Player.Caching + Integer.IsGreater(Player.CacheLevel,0)</visible>
```

Comment:
why don't you want 0% to be displayed? if for example a pvr stream stalls for some reason and cache level is zero (which is likely), you don't get the visual indicator. just a frozen picture that can mean anything.